snackutil.py

```diff
@@ -168,7 +168,8 @@ def showMessageDialog(screen, title, text):
     screen.pushHelpLine(PLEASE_WAIT_STRING)
     screen.refresh()
 
-def displayProgressDialog(screen, current, (form, t, scale), updated_text=None):
+def displayProgressDialog(screen, current, formInfo, updated_text=None):
```

Comment:
This should be `form_info` to fit the naming convention.

Also, I couldn't see where the caller was updated?

Reply:
The callers don't use named arguments they can still pass form_info in the form (form, t, scale) but instead of it being unpacked in the method definition, the tuple is assigned to form_info and then unpacked on the first line
So the callers don't need updating :)